### DISCUSSION TITLE
【基础知识分享 from chatgpt】如何理解这个等式 a+b≡c(mod p)

### DISCUSSION THREAD

**Discussion by luckyyang** [zkp-co-learning/zkp-co-learn#22] [Show and tell]
这个等式表示当整数a和b被模数p除后所得的余数之和等于c被模数p除后所得的余数时，a和b被认为是在模p意义下同余的。

具体来说，如果a和b是任意的整数，p是一个正整数，c是一个非负整数，且(a+b)÷p的余数等于c÷p的余数，则我们可以写成a+b≡c(modp)。

在数学中，"≡"这个符号表示同余关系，因此a+b≡c(modp)的意思就是"a和b在模p意义下同余于c"。同余关系在很多数学领域都有应用，例如数论、代数、计算机科学等。

**Comment by Demian101** [comment]
🆒
